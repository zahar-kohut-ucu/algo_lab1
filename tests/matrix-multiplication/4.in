16 8
25 16 12 17 93 100 72 44 
67 50 13 34 56 0 83 15 
53 8 47 2 33 5 17 23 
94 65 62 25 100 98 63 67 
58 72 72 1 88 9 22 19 
69 43 81 5 79 30 91 7 
48 15 2 11 38 6 70 39 
30 78 93 68 19 70 74 97 
34 79 73 10 72 62 43 71 
6 14 52 53 45 38 72 97 
62 76 17 8 65 14 59 83 
100 58 2 55 72 66 97 8 
100 36 100 90 62 64 14 96 
31 45 42 21 59 2 75 12 
85 58 58 35 81 69 87 88 
88 3 71 24 86 36 5 98 
8 5
33 69 61 49 8 
92 100 13 66 90 
75 76 61 68 95 
46 74 39 52 98 
87 26 93 80 11 
13 65 39 51 42 
68 46 13 11 79 
95 54 46 6 44 
