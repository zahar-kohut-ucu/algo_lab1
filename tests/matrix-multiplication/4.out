22446 20101 18637 17577 17293 
21291 19211 13833 14718 17436 
12379 11384 10825 9645 8733 
35505 35034 28359 28505 28083 
25058 21659 18600 20397 17802 
26654 24155 19926 20880 22233 
15469 12482 10146 8400 10918 
35079 35656 21090 22964 36022 
31064 28248 20883 22185 25214 
26344 21878 16852 14422 23489 
28415 23558 17295 16687 19351 
25794 27978 20020 21965 23179 
34550 36360 29038 27710 31060 
20678 17147 12949 14156 17532 
36321 34008 26245 25241 29374 
27209 23642 24649 19945 17236 
